guides/hosting/installation-updates/deployments/deployment-helper.md

```diff
@@ -0,0 +1,95 @@
+---
+nav:
+  title: Deployment Helper
+  position: 15
+
+---
+
+# Deployment Helper
+
+::: warning
+The Deployment Helper is experimental and configuration may change in the future. Please use it with caution.
+:::
+
+The Deployment Helper is a tool that unifies the steps executed after the Code has been uploaded to the server. On a traditional deployment, you would run it after the files has been uploaded, or when you are using a Containerized environment, you would run deployment helper with the new source code and then switch over the traffic.
+
+## Installing the Deployment Helper
+
+The Deployment Helper is a composer package and can be installed via composer:
+
+```bash
+composer require shopware/deployment-helper
+```
+
+and the helper can be executed via:
+
+```bash
+vendor/bin/shopware-deployment-helper run
+```
+
+## What does the Deployment Helper excactly do?
+
+The deployment helper checks for you is Shopware installed and if not, it will install it for you. It will also check if the database server is accessible and if not, it will wait until it is.
```

Comment:
```suggestion
The deployment helper checks for you if Shopware is installed and if not, it will install it for you. It will also check if the database server is accessible, and if not, it will wait until it is.
```